draft-ietf-quic-transport.md

```diff
@@ -2178,12 +2180,16 @@ before the handshake is confirmed, as defined in section 4.1.2 of {{QUIC-TLS}}.
 
 If the peer sent the disable_active_migration transport parameter, an endpoint
 also MUST NOT send packets (including probing packets; see {{probing}}) from a
-different local address to the address the peer used during the handshake. An
-endpoint that has sent this transport parameter, but detects that a peer has
-nonetheless migrated to a different remote address MUST either drop the incoming
-packets on that path without generating a stateless reset or proceed with path
-validation and allow the peer to migrate. Generating a stateless reset or
-closing the connection would allow third parties in the network to cause
+different local address to the address the peer used during the handshake. If
+the server sets the Disable Migration field in the preferred_address transport
+parameter, the client MUST NOT send packets from a different local address to
+the server's preferred address.
```

Comment:
You may want to specify that when Disable Migration is set, to migrate to an SPA of a different address family, the client MUST (SHOULD?) use a source address belonging to the same network attachment interface as the handshake address.

Reply:
Yeah, this is getting tricky. I think we can say "SHOULD use the same network interface" when changing to the server's SPA. The server will also have to do the validation dance with the client on the new client address from the SPA.

Reply:
I don't know that we want to tackle network interfaces here, that's a much larger thing. Let's keep that separate and cover the regular TP too if we decide to do that?